### PR TITLE
LPS-114280 build service

### PR DESCRIPTION
**Pull request by marcosapmf**
Related issue: https://issues.liferay.com/browse/LPS-114280

### CODE REVIEW

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/model/listener/DDMFormInstanceModelListener.java [hunk 1]
```diff
@@ -66,12 +67,11 @@ public void onAfterRemove(DDMFormInstance ddmFormInstance)
...
-				return;
-			}
-
-			_ddmFormInstanceReportLocalService.deleteDDMFormInstanceReport(
-				ddmFormInstanceReport.getFormInstanceReportId());
+			_ddmFormInstanceReportPortalExecutor.execute(
+				() ->
+					_ddmFormInstanceReportLocalService.
+						deleteDDMFormInstanceReport(
+							ddmFormInstanceReport.getFormInstanceReportId()));
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/model/listener/DDMFormInstanceModelListener.java:74]: @marcosapmf, isn't necessary do this same change on the method `onAfterCreate`?

**Comment by natocesarrego** [comment]
@brunofarache, tu se refere aos testes de Poshi, né? Porque os testes de back-end e front-end estão sendo disparados.

**Comment by brunofarache** [comment]
@natocesarrego isso, os testes de back e front já rodam no relevant.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/model/listener/DDMFormInstanceModelListener.java (hunk 1 above)
Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/model/listener/DDMFormInstanceModelListener.java:74]: @natocesarrego
No, it is not. `onAfterCreate` just creates an empty report and there is no process there. Therefore, we can keep it as synchronous :)

**Comment by natocesarrego** [comment]
Pull request submitted to https://github.com/natocesarrego/liferay-portal/pull/558. [See changes here.](https://github.com/natocesarrego/liferay-portal/compare/3ff1198df977~1...3ff1198df977) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**